packages/itwin/property-grid/scripts/run-e2e-with-docker.js

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+const path = require('path');
+const { exec } = require('child_process');
+const packageName = "property-grid"
+const dockerImageName = `${packageName}-e2e-test-image`
+const dockerContainerName = `${packageName}-e2e-test-container`
+
+const execute = command => new Promise((resolve, reject) => {
+  exec(command, (error, stdout, stderr) => {
+    if (error) {
+      console.error(`Error executing command: ${error}`);
+      return reject(error);
+    }
+    console.log(stdout);
+    console.error(stderr);
+    resolve();
+  });
+});
+
+try {
+  const currentDirectory = process.cwd();
+  // cd to the root directory from property-grid directory
+  const rootDirectory = path.resolve(currentDirectory, '../../../');
+  process.chdir(rootDirectory);
+} catch (err) {
+  console.error('Failed to change directory:', err);
+  return;
+}
+
+// Build the Docker image
+execute(`docker build -t ${dockerImageName} -f packages/itwin/${packageName}/Dockerfile .`)
+  .then(() => {
+      // Run Docker container
+      return execute(`docker run --name ${dockerContainerName} -e UPDATE_SNAPSHOTS=${process.env.UPDATE_SNAPSHOTS} ${dockerImageName}`);
+  })
+  .then(() => {
+    if (process.env.UPDATE_SNAPSHOTS) {
+      // Copy snapshots from docker container to the local repo
+      return execute(`docker cp ${dockerContainerName}:/workspaces/viewer-components-react/packages/itwin/${packageName}/src/e2e-tests/PropertyGrid.test.ts-snapshots ./packages/itwin/${packageName}/src/e2e-tests`);
```

Comment:
Can we avoid using `PropertyGrid.test.ts-snapshots` folder name? It depends on test source file name...

Reply:
Removed